arcaflow_plugin_coremark_pro/coremark_pro_plugin.py

```diff
@@ -16,10 +16,14 @@
 )
 
 
-def run_oneshot_cmd(command_list, workdir) -> str:
+def run_oneshot_cmd(command, workdir) -> str:
     try:
         cmd_out = subprocess.check_output(
-            command_list, stderr=subprocess.STDOUT, text=True, cwd=workdir
+            command,
+            stderr=subprocess.STDOUT,
+            text=True,
+            cwd=workdir,
+            shell=True,
```

Comment:
Can you add a comment about why `shell=True` is necessary?

Reply:
We found a better approach, so now it is not needed.